llpc/context/llpcCompiler.cpp

```diff
@@ -138,6 +138,9 @@ opt<unsigned> ShaderCacheMode("shader-cache-mode",
                                    "load on-disk cache for read/write, 4 - load on-disk cache for read only"),
                               init(0));
 
+// -cache-full-pipelines: Add the full pipelines to the cache.
+opt<bool> CacheFullPipelines("cache-full-pipelines", desc("Add the full pipelines to the cache."), init(true));
```

Comment:
1. Does this apply only to relocatable shader compilation? If so, can you mention this in the description?
2. Are full pipelines added to the application cache or runtime cache?
3. nit: shouldn't this be 'Add full pipelines to the cache`? I'm not good at grammar, please disregard if I got it wrong.

Reply:
1) I applies in all cases.  It is simpler to implement that way, and make the option easier to understand.
2) That part is abstracted from the place where this option is used.  It does not actually know which caches because other options affect that.  However, the wording of the description could be improved to use the plural for cache.
3) Cache is also a verb, and in this case is being used as an imperative.  This is fitting for an option name because it is giving a command to the compiler.